config/routes.rb

```diff
@@ -3,13 +3,15 @@
   mount JasmineRails::Engine => '/specs' if defined?(JasmineRails)
   mount JasmineFixtureServer => '/spec/javascripts/fixtures' if defined?(Jasmine::Jquery::Rails::Engine)
 
+  # Manually written API functions
+  post '/bot/comment/id.:format', to: 'comment#create_by_token'
```

Comment:
Definitely. This was a placeholder anyway, because I realized that there would be a few issues with the `/api` endpoint because it's already being used.